contrib/juliac.jl

```diff
@@ -6,6 +6,8 @@ module JuliaConfig
 end
 
 julia_cmd = `$(Base.julia_cmd()) --startup-file=no --history-file=no`
```

Comment:
delete this line now?

Reply:
This one is used by the precompilation process